code/src/components/ClearButton.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { todos } from '../reducers/todos'
+
+import styled from 'styled-components'
+
+const ButtonContainer = styled.div`
+ display: flex;
+ justify-content: center;
+ margin: 10px;
+`;
```

Comment:
Try to be consistent with indentation - here you have one space and on the other places you have four spaces